tests/Unit/Evolution/DiscontinuousGalerkin/Test_LiftFromBoundary.cpp

```diff
@@ -0,0 +1,258 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <algorithm>
+#include <cstddef>
+#include <type_traits>
+#include <vector>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/CoordinateMaps/Affine.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.tpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.tpp"
+#include "Domain/CoordinateMaps/Wedge2D.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Evolution/DiscontinuousGalerkin/LiftFromBoundary.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/DataStructures/MakeWithRandomValues.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/MetricIdentityJacobian.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/NormalDotFlux.hpp"
+#include "NumericalAlgorithms/LinearOperators/Divergence.hpp"
+#include "NumericalAlgorithms/LinearOperators/Divergence.tpp"
+#include "NumericalAlgorithms/LinearOperators/WeakDivergence.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+struct Var1 : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+template <size_t Dim>
+struct Var2 : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+};
+
+template <typename TagsList, size_t Dim>
+auto polynomial_volume_fluxes(
+    const tnsr::I<DataVector, Dim, Frame::Inertial>& coords,
+    const Index<Dim>& powers) noexcept {
+  Variables<db::wrap_tags_in<Tags::Flux, TagsList, tmpl::size_t<Dim>,
+                             Frame::Inertial>>
+      result(get<0>(coords).size(), 1.0);
+
+  for (size_t power_index = 0; power_index < Dim; ++power_index) {
+    for (size_t i = 0; i < Dim; ++i) {
+      if constexpr (tmpl::list_contains_v<TagsList, Var1>) {
+        get<Tags::Flux<Var1, tmpl::size_t<Dim>, Frame::Inertial>>(result).get(
```

Comment:
why not initialize `result` to zero and only compute the 0-th component of `Flux<Var1, ...>` ?